Controller/MediaController.php

```diff
@@ -134,12 +134,29 @@ public function liipImagineFilterAction($path, $filter)
 
         $image = $this->get('liip_imagine')->open($tmpFile);
 
-        $response = $this->get('liip_imagine.filter.manager')->get($this->get('request'), $filter, $image, $path);
+        $response = $this->get('liip_imagine.filter.manager')->get($this->getRequest(), $filter, $image, $path);
 
         if ($targetPath) {
             $response = $this->get('liip_imagine.cache.manager')->store($response, $targetPath, $filter);
         }
 
         return $response;
     }
+    
+    
+    /**
+     * 
+     * @return Request
+     * 
+     * NEXT_MAJOR : Return $this->get('request_stack')->getCurrentRequest() only
+     * 
+     */
+    public function getRequest()
```

Comment:
In this case you should not have to rely on this. This method is for BC. For you case, just use the appropriate method according to your installed Symfony version.